pkg/dpll/dpll.go

```diff
@@ -621,77 +620,81 @@ func (d *DpllConfig) MonitorDpll() {
 // stateDecision
 func (d *DpllConfig) stateDecision() {
```

Comment:
I have a general concern about our approach in determining the clock class based on independent events when having multiple DPLLs. How do we prevent a situation like this:
1. "DPLL A" signals holdover (misconfiguration or HW failure).
2.  We enter T-GM holdover or freerun
3. "DPLL B" reports that everything is fine
4. We close the holdover and enter Locked state

Is there a mechanism to look at the whole picture and not at its parts?